test/onnx/test_fx_op_consistency.py

```diff
@@ -162,6 +162,54 @@
     ]
 )
 
+
+# NOTE: For ATen signature modifications that will break ONNX export,
+# use **xfail_onnxscript_forward_compatibility** and **skip_torchlib_forward_compatibility** instead of xfail or skip
+# to make the signal apparent for maintainers.
+def xfail_onnxscript_forward_compatibility(
```

Comment:
Maybe in the future we also want to version it against onnxscript? Just some thoughts.

Reply:
What kind of versioning do you have in mind?